src/main.rs

```diff
@@ -393,6 +394,9 @@ fn main() -> anyhow::Result<()> {
         log::info!("Telemetry reporting disabled");
     }
 
+    // Setup subscribers to listen for issue-able events
+    issues_setup::setup_subscribers(dispatcher_arc.clone());
```

Comment:
Can we maybe protect the feature behind a configuration flag?
It would be great to be able to disable it if necessary.

Reply:
Yes, one of the advantages of having the event broker. I'll leave it enabled by default

Reply:
Sorry, I meant a runtime configuration with a new config field.
The idea being that we can experiment with it easily and disable it if it creates problems.
WDYT?